libcudacxx/include/cuda/std/__functional/ranges_operations.h

```diff
@@ -29,6 +29,8 @@
 _LIBCUDACXX_BEGIN_NAMESPACE_RANGES
 _LIBCUDACXX_BEGIN_NAMESPACE_RANGES_ABI
 
+// TODO(bgruber): do we need to specialize proclaims_copyable_arguments here as well?
+
 struct equal_to
```

Comment:
@miscco should we proclaim all the range operators as well?

Reply:
yes